mailet/standard/src/test/java/org/apache/james/transport/mailets/AmqpForwardAttributeTest.java

```diff
@@ -172,10 +172,10 @@ public void serviceShouldNotUseConnectionWhenNoAttributeInMail() throws Exceptio
     }
 
     @Test
-    public void serviceShouldThrowWhenAttributeContentIsNotAMap() throws MessagingException {
+    public void serviceShouldThrowWhenAttributeContentIsNotAMapAListOrAString() throws MessagingException {
```

Comment:
this test name is bad at this state of the pr (because string is still not supported), but that's not a big deal

Reply:
Yes, I hesitate to squash the two commits for that ;)